### ISSUE TITLE
Some question about the performance of VectorNet

### ISSUE THREAD

**Issue by Hugh-Cai** [xk-huang/yet-another-vectornet#32]
Hi, 
Whether are there any key points or tricks when re-implement the model? I tried to implement it without `torch_geometric`, but have some problems:
1. the training loss reduces slowly
2. the predictor produces almost the same offset trajectories for all scenario (the encoder and the decoder is the same as the original paper)
![Figure_0](https://user-images.githubusercontent.com/56687411/203758739-0d8a165a-186c-40d3-a6ea-95d4e38c4f3d.png)
![Figure_1](https://user-images.githubusercontent.com/56687411/203758767-2025283a-7169-4779-9b8d-809aae56e004.png)
